titan/output.py

```diff
@@ -22,7 +22,7 @@ def setup_aggregates(
     Attributes are classes defined in params, the items counted are:
 
     * "agents"
-    * "deaths_hiv"
+    * "[exit_type]_hiv"
```

Comment:
I think these are `exit class` and not `exit type` - we should be really careful in the usage here since the two are similar, but different.